site/config/init.php

```diff
@@ -4,3 +4,8 @@
 include('include/db_query.php'); //this should happen right after config so other functions have access to the database
 include('include/header.php');
 include('include/footer.php');
+include('include/FAQFunction.php');
```

Comment:
This is ok, but I want to make sure you're aware that it's normal to define multiple functions in one file. For example, you might just have a `header.php` file which could define the header and footer functions. These other files you've made are all different parts of the site so it's probably fine to have them as different files, but as you expand on this project, you'll want multiple functions in a single file. Like, `FAQFunction` might have a `CreateFaqItem` function, `GetFaqItem`, `GetAllFaqItems`, etc.

Side note: we didn't used to do it this way so you'll see examples in our main code base that don't follow this rule, but our current way of capitalizing things with acronyms like "FAW" is to only capitalize the first letter because otherwise it's hard to tell when the next word begins. So it'd be `FaqFunction` rather than `FAQFunction`.